### PR TITLE
Added schema explanations for the input parameters in app.yaml.

### PR DESCRIPTION
**Pull request by omarabed15**

### CODE REVIEW

**Comment by gregelin** [comment]
@omarabed15 Could you add to the description of Inputs a reference that this feature was introduced into version 0.9.1.50.1? could be a short, "Version 0.9.1.50.1 introduced ..." or "Since version 0.9.1.50.1.

**Review by gregelin — CHANGES_REQUESTED** [review]
Please add reference to fact "inputs" first available in a particular version/release of application.

**Comment by omarabed15** [comment]
> Please add reference to fact "inputs" first available in a particular version/release of application.

@gregelin I added a statement to that effect.

File: source/authoring-guide/modules-questions-inputs-documents-yaml-reference.rst [hunk 1]
```diff
@@ -169,6 +169,34 @@ e.g.:
              q1: desktop
              q2: My Secure Tool
 
+Inputs
```
Comment (davidpofo) [inline source/authoring-guide/modules-questions-inputs-documents-yaml-reference.rst:172]: great stuff thanks for adding @omarabed15

**Review by davidpofo — APPROVED** [review]

**Pull request merged**